README.md

```diff
@@ -10,15 +10,15 @@ for developing state-of-the-art deep learning models on a wide variety of lingui
 ## Quickstart
 
 The fastest way to get an environment to run AllenNLP is with Docker.  Once you have [installed Docker](https://docs.docker.com/engine/installation/)
-just run `docker run -it --rm allennlp/allennlp` to get an environment that will run on either the cpu or gpu.
+just run `docker run -it --rm allennlp/allennlp:v0.3.0` to get an environment that will run on either the cpu or gpu.
 
 Now you can do any of the following:
 
 * Run a model on example sentences with `python -m allennlp.run predict`.
 * Start a web service to host our models with `python -m allennlp.run serve`.
 * Interactively code against AllenNLP from the Python interpreter with `python`.
 
-You can also install via the `pip` package manager or by cloning this repository into a Python 3.6 virtualenv.
+You can also install via the `pip` package manager or by cloning this repository into a Python 3.6 Conda environment.
```

Comment:
I mean, it should work with conda or virtualenv, I would probably change this to

or by cloning this repository and creating a clean Python 3.6 environment.

(you don't really clone "into" an environment)